crates/starknet/src/lib.rs

```diff
@@ -238,6 +250,11 @@ mod tests {
             seed: 123,
             total_accounts: 3,
             predeployed_accounts_initial_balance: 100.into(),
+            host: String::from("127.0.0.1"),
+            port: 5050,
```

Comment:
Can be extracted in default consts and reused in cli.rs

Reply:
I'm not sure how to do it :( a little help?

Reply:
solved in recent commits